service/operator/resource/service/service.go

```diff
@@ -0,0 +1,287 @@
+package service
```

Comment:
Before there have been both resources together in the implemented `Operator` interface. Now the service resource is separated into its own `Resource` implementation. Besides moving code into its own package and adapting to the import changes of the k8s client due to vendor updates nothing changed. Tests have been adapted as well.